src/Shared.js

```diff
@@ -6,8 +6,8 @@
 
 module.exports = {
 
-	analytics: require('hyper-analytics'), // npm
-    //analytics: require('../../hyper-analytics/src'), // developers
+	// analytics: require('hyper-analytics'), // npm
```

Comment:
Undo this change. This file should never be merged in an altered state. Meaning, build depends on latest hyper-analytics being fully merged and published to npm.